client/src/hooks/helpers/useTrade.tsx

```diff
@@ -141,7 +141,7 @@ export function useGetMyOffers(): MarketInterface[] {
 
   const { computeTrades } = useTrade();
 
-  const { realmEntityId } = useRealmStore();
+  const { realmEntityId } = useUIStore();
```

Comment:
need to use the state.realmEntityId EVERYTIME you access zustand store, it was ok for useRealmStore because not lots of values inside but not ok for useUIStore and in general we should just always access through state.realmEntityId